packages/dev/__tests__/cli-test.ts

```diff
@@ -0,0 +1,154 @@
+import childProcess from "node:child_process";
+import os from "node:os";
+import path from "node:path";
+import util from "node:util";
+import fse from "fs-extra";
+
+let execFile = util.promisify(childProcess.execFile);
+
+const TEMP_DIR = path.join(
+  fse.realpathSync(os.tmpdir()),
+  `rr-tests-${Math.random().toString(32).slice(2)}`
+);
+
+jest.setTimeout(30_000);
+beforeAll(async () => {
+  await fse.remove(TEMP_DIR);
+  await fse.ensureDir(TEMP_DIR);
+});
+
+afterAll(async () => {
+  await fse.remove(TEMP_DIR);
+});
+
+async function execCli(
+  args: Array<string>,
+  options: Exclude<Parameters<typeof execFile>[2], null | undefined> = {}
+) {
+  if (process.platform === "win32") {
+    let cp = childProcess.spawnSync(
+      "node",
+      [
+        "--require",
+        require.resolve("esbuild-register"),
+        path.resolve(__dirname, "../cli.ts"),
+        ...args,
+      ],
+      {
+        cwd: TEMP_DIR,
+        ...options,
+        env: {
+          ...process.env,
+          NO_COLOR: "1",
+          ...options.env,
+        },
+      }
+    );
+
+    return {
+      stdout: cp.stdout?.toString("utf-8"),
+    };
+  } else {
+    let result = await execFile(
+      "node",
+      [
+        "--require",
+        require.resolve("esbuild-register"),
+        path.resolve(__dirname, "../cli.ts"),
+        ...args,
+      ],
+      {
+        cwd: TEMP_DIR,
+        ...options,
+        env: {
+          ...process.env,
+          NO_COLOR: "1",
+          ...options.env,
+        },
+      }
+    );
+    return {
+      ...result,
+      stdout: result.stdout.replace(TEMP_DIR, "<TEMP_DIR>").trim(),
+    };
+  }
+}
+
+describe("rr CLI", () => {
+  describe("the --help flag", () => {
+    it("prints top-level help info", async () => {
+      let { stdout } = await execCli(["--help"]);
+      let version = require("../package.json").version;
+      expect(stdout.trim().replace(version, "[VERSION]"))
+        .toMatchInlineSnapshot(`
+        "rr/[VERSION]
+
+        Usage:
+          $ rr <command> [options]
+
+        Commands:
+          vite:dev [root]    build for production
```

Comment:
Can these just be `dev` and `build` without the `vite:` prefix?

Reply:
Updated 👍